odh/base/monitoring/overrides/grafana-operator/overlays/dashboards/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
```

Comment:
What is interesting to notice here @HumairAK @hemajv you don't even need the `base` in the resources in here, despite it being an overlay. The way how kfctl folds and combines overlays and writes a new `kustomization.yaml`  for you from scratch is very magical. :slightly_smiling_face: